### DISCUSSION TITLE
xmake service -vD wait a input

### DISCUSSION THREAD

**Discussion by yujunting** [xmake-io/xmake#5636] [Q&A]
远程主机(win)启动了一个xmake service -vD
本地使用 xmake service --connect --distcc --ccache命令连接成功后
使用xmake命令后，本地显示：
<remote_build_client>: run 'xmake' in 10.0.13.94:9691 ..
就卡住了
远程主机提示：
note: xmake.lua not found, try generating it (pass -y or --confirm=y/n/d to skip confirm)?
please input: n (y/n)
此时输入y/n都无效没反应
想知道应该怎么处理？

**Comment by yujunting** [comment]
v2.9.5版本依旧有这个问题

**Comment by waruqi** [comment]
提供完整可复现的 命令执行和 两端 -vD logs

另外，本地确认下 是否在当前项目目录下执行，是否存在 xmake.lua ，执行过程给下完整 logs 

以目前的信息看，你本地执行 xmake 命令的目录下 没有有效的 xmake.lua 文件和工程存在。

**Comment by yujunting** [comment]
先关掉这个问题了，后面再试又没有了
